### PR TITLE
URL function complete

### PR DESCRIPTION
**Pull request by kjy95**
I done the read URL function. You can put URL into the TextField. By doing so TextView show you the site's text information. So you or blind person can hear the site's information. It is specially useful to blind.

### CODE REVIEW

File: Say/Base.lproj/Main.storyboard [hunk 1]
```diff
@@ -1,5 +1,5 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<document type="com.apple.InterfaceBuilder3.Cocoa.Storyboard.XIB" version="3.0" toolsVersion="11201" systemVersion="16A323" targetRuntime="MacOSX.Cocoa" propertyAccessControl="none" useAutolayout="YES" initialViewController="B8D-0N-5wS">
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (youknowone) [inline Say/Base.lproj/Main.storyboard]: We don't need to update storyboard file for this feature.

File: Say/ViewController.swift [hunk 2]
```diff
@@ -131,35 +131,24 @@ class ViewController: NSViewController {
     
     @IBAction func selectText(_ sender: NSTextField) {
         if let url = URL(string: sender.stringValue) {
+            let instapaper = "https://www.instapaper.com/text?u="
+            let onlyText = URL(string:"\(instapaper)\(url)")
```
Comment (youknowone) [inline Say/ViewController.swift]: The name `onlyText` looks like `String` object. I recommend to use `textURL` or `instapaperURL`.

File: Say/ViewController.swift [hunk 3]
```diff
@@ -131,35 +131,24 @@ class ViewController: NSViewController {
...
             // if URL format is right
-            if let data = NSData.init(contentsOf: url) {
+            if let data = NSData(contentsOf: onlyText!) {
```
Comment (youknowone) [inline Say/ViewController.swift]: Move `!` operator to upper line where the URL is built.

About the operator `!`, it casts optional type to value type. So in this case, `onlyText` is `URL?`. But if you are sure the URL is always not-null, the variable is better to be value there as `URL` and it doesn't need to be casted in if-statement.

**Review by youknowone — CHANGES_REQUESTED** [review]

File: Say/ViewController.swift (hunk 2 above)
Comment (kjy95) [reply Say/ViewController.swift]: Thank you! I'll change it!

**Pull request merged**